homeassistant/components/peco/__init__.py

```diff
@@ -16,7 +17,10 @@ async def async_setup_entry(hass: HomeAssistant, entry: ConfigEntry) -> bool:
     """Set up PECO Outage Counter from a config entry."""
 
     hass.data.setdefault(DOMAIN, {})
-    hass.data[DOMAIN][entry.entry_id] = peco
+    hass.data[DOMAIN][entry.entry_id] = {
+        "api": peco,
+        "websession": aiohttp.ClientSession(),
```

Comment:
you should use `from homeassistant.helpers.aiohttp_client import async_get_clientsession` or the httpx equivalent https://github.com/home-assistant/core/blob/b05b4c4b3836cce84a63de0f5746ed231a1166cf/homeassistant/helpers/httpx_client.py#L26 for this